backend/custom_models/migrations/0013_upload_background_tree_view_info.py

```diff
@@ -0,0 +1,636 @@
+# Generated by Django 2.1.7 on 2019-05-30 14:07
+# Edited by Francis Normand to upload the background tree view
+# content (one in Organizational Structure section and one in Team
+# Information section) of the image descriptions for the sample test
+
+from django.db import migrations
+
+
+def upload_background_tree_view_info(apps, schema_editor):
+    # get models
+    language = apps.get_model("custom_models", "Language")
+    item_type = apps.get_model("custom_models", "ItemType")
+    item = apps.get_model("custom_models", "Item")
+    item_text = apps.get_model("custom_models", "ItemText")
+
+    # get db alias
+    db_alias = schema_editor.connection.alias
+
+    # lookup languages; do not use bulk_create since we need these objects later on
+    l_english = (
+        language.objects.using(db_alias)
+        .filter(ISO_Code_1="en", ISO_Code_2="en-ca")
+        .last()
+    )
+    l_french = (
+        language.objects.using(db_alias)
+        .filter(ISO_Code_1="fr", ISO_Code_2="fr-ca")
+        .last()
+    )
+    background_id = (
+        item_text.objects.using(db_alias)
+        .filter(text_detail="Background", language=l_english)
```

Comment:
wondering if we should get the emibSampleTest item, then get the background that is the child of that item... idk how much migrating back and forth we are actually likely to  be doing with these early migrations, but something to consider...

Reply:
because this will return any item (be it a question, answer, test, background, etc) who's english text reads "Background", and then get the last instance.. that's my biggest hesitation...

Reply:
Sorry, I didn't even think of it till I was actually looking at the code just now....

Reply:
Added in commit #https://github.com/code-for-canada/project-thundercat/pull/409/commits/1aaa156edc0ce576d6dee0418f832b3f6436486a

Reply:
fixed (saw it offline)